src/app/index.module.ts

```diff
@@ -219,12 +247,12 @@ angular.element(document).ready(() => {
     cheBranding.ready.then(() => {
       (angular as any).resumeBootstrap();
     });
-  }).catch((error: string) => {
-    console.error(`Can't GET "/api". ${error ? 'Error: ' : ''}`, error);
-    if (!hasCertificateError) {
```

Comment:
Do we still how TLS certificate issue error message?

Reply:
correct